.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Release workflow
+jobs:
+  build-phar:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
please use `@v2` rather than `@master` to be more reliable in case this action releases a new major version in the future (same for setup-php below).